repo/nirepo.yml

```diff
@@ -0,0 +1,1590 @@
+    delete:
+      x-ni-privilege: Write
+      x-ni-operation: deletePackage
+      tags: [packages]
+      summary: Remove a package.
+      description: Remove a package.
+      responses:
+        202:
+          description: OK
+          headers:
+            Location:
+              type: string
+              description: Contains the relative URI of the job that was created for tracking the status of the operation. Execute a GET against this URI to get information about the status of the operation.
+          schema:
+            $ref: '#/definitions/JobID'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/upload-packages:
+    post:
+      x-ni-privilege: Write
+      x-ni-operation: uploadPackage
+      tags: [packages]
+      summary: Upload packages to the pool
+      description: Asynchronously uploads packages to the service's package pool. Returns an array of IDs of resource which can be queried to track the status of each upload operation.
+      consumes:
+        - multipart/form-data
+      parameters:
+        - in: query
+          name: shouldOverwrite
+          type: boolean
+          description: A query parameter that determines if existing packages with the same filename as a package to be added to the pool should be overwritten. Defaults to false.
+          required: false
+        - in: formData 
+          name: filename
+          type: file
+          required: true
+          description: The name of this parameter is the name of the file that will be uploaded. If the file you upload is test.ipkg, put instead of filename that name.
+        - in: header
+          name: X-File-Name
+          type: string
+          required: false
+        - in: header
+          name: X-File-Size
+          type: string
+          required: false
+        - in: header
+          name: X-File-Type
+          type: string
+          required: false
+      responses:
+        202:
+          description: OK
+          headers:
+            Location:
+              type: string
+              description: Contains the relative URI of the job that was created for tracking the status of the operation. Execute a GET against this URI to get information about the status of the operation.
+          schema:
+            type: object
+            properties:
+              jobIds:
+                type: array
+                items:
+                    type: string
+                    example: '5c66b6bd781e2b2ce42ca818'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/get-package-import-descriptors:
+    post:
+      x-ni-privilege: Read
+      x-ni-operation: getPackageImportDescriptors
+      tags: [packages]
+      summary: Get package import descriptors
+      description: Route for getting a description of the packages in a remote feed.
```

Comment:
I'm not sure. I saw it on nitalk, but there is not such implementation in the webservice. That route does not seem to exist.

Reply:
Oh ok, maybe it was never implemented. If the selective import route isn't implemented, I would suggest against documenting this route, as I don't think it can be used for anything without that route.

Reply:
Then we should remove it. Otherwise, I think the process is that since it is a public route it should be documented.